### PR TITLE
SC-1325

### PR DESCRIPTION
**Pull request by anmol2302**
<b>Ticket Ref:</b> [SC-1325](https://project-sunbird.atlassian.net/browse/SC-1325)
<b>DOC :</b> [Onboarding APIs Reference](https://docs.google.com/spreadsheets/d/15fQRG1D8D214XhbLh-RKv078SJ2Au1_nXDMbsbUnpLc/edit#gid=772094723)

1: /v2/user/bulk/upload API onboarding

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -4494,3 +4494,19 @@ kong_apis:
...
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist: userAdmin
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
```
Comment (manzarul) [inline ansible/roles/kong-api/defaults/main.yml]: i think change this to "small_rate_limit_per_hour"

Comment (anmol2302) [reply ansible/roles/kong-api/defaults/main.yml]: ok

File: ansible/roles/kong-api/defaults/main.yml [hunk 2]
```diff
@@ -4494,3 +4494,19 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+  - name: v2UserBulkUpload
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml]: Change the API name to UserBulkUploadv2, let the APIs following same naming convention

Comment (anmol2302) [reply ansible/roles/kong-api/defaults/main.yml]: DONE

**Pull request merged**